packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_widgets/flutter_widgets.dart';
+
+/// Convenience [Divider] with [Padding] to fit in better with forms.
```

Comment:
nit: 'to fit in better with forms' --> 'that provides a good divider within a form.

Reply:
Done.